app/earth/index.html

```diff
@@ -46,96 +30,27 @@
-     <script src="app/collections/Configurations.js"></script>
-     <script src="app/collections/Community.js"></script>
-     <script src="app/collections/MailUnopened.js"></script>
-     <script src="app/collections/siteFeedbacks.js"></script>
-     <script src="app/collections/Reports.js"></script>
-     <script src="app/collections/Publication.js"></script>
-     <script src="app/collections/CourseRequest.js"></script>
-     <script src="app/collections/ResourceRequest.js"></script>
-     <script src="app/collections/NationReportComments.js"></script>
-     <script src="app/collections/Courses.js"></script>
-     <script src="app/collections/Resources.js"></script>
-     <script src="app/collections/Members.js"></script>
-     
-     
      
      <!-- Views  -->
 
-     <script src="app/views/MemberForm.js"></script>
      <script src="app/views/NationsTable.js"></script>
      <script src="app/views/NationForm.js"></script>
-     <script src="app/views/CommunityForm.js"></script>
-     <script src="app/views/ConfigurationView.js"></script>
      <script src="app/views/navBarView.js"></script>
      <script src="app/views/MemberLoginForm.js"></script>
-
-     <script src="app/views/NationTable.js"></script>
      <script src="app/views/NationRow.js"></script>
 
-
-    <script src="app/views/Dashboard.js"></script> 
-    <script src="app/views/siteFeedback.js"></script>
-    <script src="app/views/siteFeedbackPage.js"></script>
-    <script src="app/views/siteFeedbackPageRow.js"></script>
-    <script src="app/views/ReportsRow.js"></script>
-    <script src="app/views/Publication.js"></script>
-    <script src="app/views/PublicationTable.js"></script>
-    <script src="app/views/PublicationRow.js"></script>
-    <script src="app/views/ReportsTable.js"></script>
-    <script src="app/views/ReportForm.js"></script>
-  	<script src="app/views/PublicationResourceTable.js"></script>
-    <script src="app/views/PublicationResourceRow.js"></script>
-    <script src="app/views/PublicationForm.js"></script>
-    <script src="app/views/RequestsTable.js"></script>
-    <script src="app/views/RequestRow.js"></script>
-	<script src="app/views/NationReportCommentView.js"></script>
-	<script src="app/views/listCommunityView.js"></script>
-  	<script src="app/views/CourseView.js"></script>
-  	<script src="app/views/CoursesTable.js"></script>
-  	<script src="app/views/CourseRow.js"></script>
-  	<script src="app/views/listSyncDbView.js"></script>
-  	<script src="app/views/courseSeach.js"></script>
-  	<script src="app/views/PublicationCoursesTable.js"></script>
-  	<script src="app/views/PublicationCourseRow.js"></script>
-  <script src="app/views/NationApplicationView.js"></script>
-  <script src="app/views/PendingNationApplicationsView.js"></script>
-  <script src="app/views/NationApplicationRow.js"></script>
-  <script src="app/views/NationApplicationsTable.js"></script>
-
-  	
   	<script src="app/Router.js"></script>
      App.Views.CourseRow
```

Comment:
I believe this line of code does not do anything. We have removed it on the index.html of MyApp